mozregression/launchers.py

```diff
@@ -289,13 +289,13 @@ class FennecLauncher(Launcher):
 
     @classmethod
     def check_is_runnable(cls):
-        # ADBHost().devices() seems to raise OSError when adb is not
-        # installed and in PATH. TODO: maybe fix this in mozdevice.
+        # ADBHost().devices() seems to raise ADBError
+        # when adb is not executable or not in PATH.
         try:
             devices = ADBHost().devices()
-        except OSError:
+        except ADBError:
             raise LauncherNotRunnable("adb (Android Debug Bridge) is not"
-                                      " installed or not in the PATH.")
+                                      " executable or not in the PATH.")
```

Comment:
So, first, avoid creating a new PR - just add new commits and push them in this PR.
Then, you should not write a string to explain the failure, because we want the string contained inside the ADBError exception that we intercept.

In other words, the idea is to create a new error that should contain the same text as the first error.